sdk/contentsafety/ai-content-safety-rest/src/models.ts

```diff
@@ -1,31 +1,43 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-/** The analysis request of the text. */
+/** The text analysis request. */
 export interface AnalyzeTextOptions {
-  /** The text needs to be scanned. We support at most 1000 characters (unicode code points) in text of one request. */
+  /** The text needs to be analyzed. We support a maximum of 10k Unicode characters (Unicode code points) in the text of one request. */
   text: string;
-  /** The categories will be analyzed. If not assigned, a default set of the categories' analysis results will be returned. */
+  /** The categories will be analyzed. If they are not assigned, a default set of analysis results for the categories will be returned. */
   categories?: string[];
   /** The names of blocklists. */
   blocklistNames?: string[];
   /** When set to true, further analyses of harmful content will not be performed in cases where blocklists are hit. When set to false, all analyses of harmful content will be performed, whether or not blocklists are hit. */
-  breakByBlocklists?: boolean;
+  haltOnBlocklistHit?: boolean;
+  /**
+   * This refers to the type of text analysis output. If no value is assigned, the default value will be "FourSeverityLevels".
+   *
+   * Possible values: FourSeverityLevels, EightSeverityLevels
+   */
+  outputType?: string;
```

Comment:
why the possible values for outputType in AnalyzeTextOptions and AnalyzeImageOptions are different ? And what would happen if we don't pass the outputType in the client side ?

Reply:
Because the models behind these two analyze APIs currently support different output types, image doesn't support outputting 8 level severity yet.
This parameter is not required. Request defaults to 4 level severity if this parameter is not carried in the body.